spec/requests/distributions_by_county_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "rails_helper"
+
+RSpec.describe "DistributionsByCounties", type: :request do
+  let(:default_params) do
+    {organization_id: @organization.to_param}
+  end
+
+  include_examples "distribution_by_county"
+
+  context "While not signed in" do
+    it "redirects for authentication" do
+      get distributions_by_county_report_path(default_params)
+      expect(response).to be_redirect
+    end
+
+    context "While signed in as bank" do
+      before do
+        sign_in(@user)
+      end
+
+      describe "show" do
+        it "returns http success" do
+          get distributions_by_county_report_path(default_params)
+          expect(response).to have_http_status(:success)
+          expect(response.body).to include("Estimated Distributions by County for ")
```

Comment:
Um - this doesn't have any actual data that's it checking for. Can we validate that we're at least viewing the page for the right distribution etc?

Reply:
Removing the plain jane test, as, if it would fail, the other tests will also fail.